xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxTradeServiceRaw.java

```diff
@@ -57,6 +58,38 @@ public String placeLimitOrderForSubaccount(String subaccount, LimitOrder limitOr
     }
   }
 
+  public FtxResponse<FtxOrderDto> modifyFtxOrder(
+      String subaccount, String orderId, FtxModifyOrderRequestPayload payload)
+      throws FtxException, IOException {
+    try {
+      return ftx.modifyOrder(
+          exchange.getExchangeSpecification().getApiKey(),
+          exchange.getNonceFactory().createValue(),
+          signatureCreator,
+          subaccount,
+          orderId,
+          payload);
+    } catch (FtxException e) {
+      throw new FtxException(e.getMessage());
```

Comment:
remove try catch

let the original exception get out

same goes for the code below